provisioning/support.yaml

```diff
@@ -0,0 +1,2 @@
+support_bundle:
+  - /opt/seagate/cortx/hare/bin/hare_setup support_bundle

```

Comment:
Are you sure that this is sufficient to say that we're supporting the support bundle framework?

I honestly tried to find this information in https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/316735509/Support+Bundle+Framework but I can't find the requirements for end components.

Reply:
I spoke with Swapnil and Papan Kumar Singh about the support bundle requirements. And both of them suggested referring to this [EOS-20508](https://jts.seagate.com/browse/EOS-20508) this ticket test our changes with support_bundle. In this task, we have to add support.yaml file and confirmed that it should present at `/opt/seagate/cortx/hare/conf/support.yaml` on deployed setup.
cc. @SwapnilGaonkar7 @vaibhavparatwar